src/main/webapp/help-continuationCondition.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Define how to decide the status of the whole MultiJob Phase, based on the statuses of phase's sub jobs. If the condition failed the build will fail.
+    Define what the status of current phase will abort calculating of other MultiJob phases.

```

Comment:
Please, revert this to the original. We should not make multiple unrelated fixes/changes in one pull request.